devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -210,7 +210,7 @@ def configure() {
 
 	def cmds
 	if (supportsLiftPercentage()) {
-		cmds = zigbee.configureReporting(CLUSTER_WINDOW_COVERING, ATTRIBUTE_POSITION_LIFT, DataType.UINT8, 0, 600, null)
+		cmds = zigbee.configureReporting(CLUSTER_WINDOW_COVERING, ATTRIBUTE_POSITION_LIFT, DataType.UINT8, 2, 600, null)
```

Comment:
Did you try a value of `1`?

Reply:
Yes, but it didn't make much difference, that's why it was changed to 2 seconds.